docs/nlu/choosing-a-pipeline.rst

```diff
@@ -276,8 +276,17 @@ default is to use a simple whitespace tokenizer:
     - name: "CRFEntityExtractor"
     - name: "EntitySynonymMapper"
     - name: "CountVectorsFeaturizer"
+    - name: "CountVectorsFeaturizer"
+      analyzer: "char_wb"
+      min_ngram: 1
+      max_ngram: 4
     - name: "EmbeddingIntentClassifier"
 
+The pipeline uses two instances of ``CountVectorsFeaturizer``. The first one 
+featurizes text based on words. The second one featurizes text based on character 
+n-grams, preserving word boundaries. We empirically found the second featurizer 
+to be more powerful.
```

Comment:
if the second is more powerful then why are we keeping the first one? I guess what you wanna say is that adding the second featurizer makes the whole intent classification more robust (to typos?)

Reply:
Well, strictly speaking, the 1st featuriser isn't needed, but we are keeping it to make the featurisation more robust (importantly, keeping it doesn't make things worse, it can only make them better).

Reply:
sounds good now 👍

Reply:
@wochinge I've changed the wording.